tests/time_tests/test_runner/conftest.py

```diff
@@ -93,12 +93,12 @@ def pytest_addoption(parser):
         '--db_metadata',
         type=str,
         default="{}",
-        help='add extra commit information, json formatted')
+        help='path to JSON-formatted file to extract additional information and submit to database')
     db_args_parser.addoption(
         '--manifest',
         type=Path,
         required=is_db_used,
-        help='extract commit information from build manifest')
+        help='path to build manifest to extract commit information and submit to database')
```

Comment:
"and submit to database" is weekly related to those two options. Can we remove this clause?

Reply:
Done